src/app/components/header/header.component.scss

```diff
@@ -0,0 +1,70 @@
+.header{
+    width: 100%;
+    height: 72px;
```

Comment:
Use rem as spacing related unit. Also introduce different level's sass variables for spacing and colors.

Reply:
Resolve same in other places.